client/blocks/reader-unsubscribed-feeds-search-list/reader-unsubscribed-feeds-search-list.tsx

```diff
@@ -7,7 +7,7 @@ import { SOURCE_SUBSCRIPTIONS_SEARCH_RECOMMENDATION_LIST } from 'calypso/landing
 import './style.scss';
 
 const ReaderUnsubscribedFeedsSearchList = () => {
-	const { feedItems, searchQueryResult } = Reader.useUnsubscribedFeedsSearch() ?? {};
```

Comment:
While unsubcribing there were a flash of no results so removed it to make sure that we always return results even when unsubscribtion is in progress.

Reply:
Apart from above reason, now we are also getting `feedItems` from parent which are filtered to avoid duplications between subscriptions table and this list.